### DISCUSSION TITLE
How to track the login activity of any user through API

### DISCUSSION THREAD

**Discussion by himankgupta81** [community/community#24351] [Apps, API and Webhooks]
<p>Is there any way we can track the login activity of any user through API.<br>
We need to know when was the last time a user logged in to their GitHub account.</p>

**Comment by billythekid** [comment]
<p>Hi there,</p>
<p>There’s no result in the REST API specifically for the login event, but you can <a href="https://docs.github.com/en/free-pro-team@latest/rest/reference/users#get-a-user" rel="noopener nofollow ugc">get general (public facing) user information</a> at this address:</p>
<p><a href="https://api.github.com/users/billythekid" rel="noopener nofollow ugc">https://api.github.com/users/[USERNAME]</a></p>
<p>If you’re using the GRAPHQL API, you’ll want to check out this page for what information is available:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/graphql/reference/objects#user" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/graphql/reference/objects#user" target="_blank" rel="noopener nofollow ugc">Objects - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Cheers,<br>
Billy</p>
